packages/zeebe-element-templates-json-schema/test/spec/validationSpec.js

```diff
@@ -389,6 +389,16 @@ describe('validation', function() {
 
       testTemplate('placeholder-invalid-type');
     });
+
+
+    describe('engines', function() {
+
+      testTemplate('engines');
+
+      testTemplate('engines-invalid');
+
+      testTemplate('engines-invalid-version', null, true);
```

Comment:
This executes an _only_ test.

```suggestion
      testTemplate('engines-invalid-version');
```